<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMCentralWidget</class>
 <widget class="QWidget" name="TMCentralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="TMMessageWindow" name="messageWindow">
      <attribute name="title">
       <string>Messages</string>
      </attribute>
     </widget>
     <widget class="TMLogWindow" name="logWindow">
      <attribute name="title">
       <string>Logs</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TMMessageWindow</class>
   <extends>QWidget</extends>
   <header>tmmessagewindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TMLogWindow</class>
   <extends>QWidget</extends>
   <header>tmlogwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
